tests/utils_test.py

```diff
@@ -608,13 +608,19 @@ def test_get_mtab():
 
 def test_get_file_device_path():
     # Arrange
+    test_symlink = "/tmp/test_symlink"
+    os.symlink("/foobar/test", test_symlink)
 
     # Act
-    result = utils.get_file_device_path("/etc/os-release")
+    result = utils.get_file_device_path(test_symlink)
+
+    # Cleanup
+    os.remove(test_symlink)
```

Comment:
I wonder here, could we replace this with the `tmpdir` fixture? This means that you won't have to do the cleanup manually.

I seem to remember I ran into some issues when trying to use `tmpdir` in Cobbler, though I don't remember what exactly that was (something permission related IIRC). So if that doesn't work, this is good enough IMHO.

Reply:
It seems to work and this is what Pablo committed to `main`. As such I will leave it.